### DISCUSSION TITLE
What needs to be changed in the QEMU arguments for the dev branch?

### DISCUSSION THREAD

**Discussion by gentoocat** [ChefKissInc/Inferno#13] [Q&A] [answered]
Hello, what qemu command i should use for `dev` branch?

I used guide from [this](https://github.com/TrungNguyen1909/qemu-t8030/wiki/Bringing-up-the-emulator#auto-boot)

qemu says that

`qemu-system-aarch64: Property 't8030-machine.trustcache-filename' not found`

Can you give me QEMU command that works? Or if you can, please write some mini-guide about running in qemu.

I will be very grateful for your help.

**Comment by Arbifoxx** [comment]
I'm having this same issue, I'm glad it's not just me. Anyone know why?

**Comment by Arbifoxx** [comment, marked as answer]
https://github.com/ChefKissInc/qemu-apple-silicon/discussions/16#discussioncomment-8720728 @gentoocat
